src/pattern.jl

```diff
@@ -258,7 +258,7 @@ function local_jacobian_pattern(f!, y, x, ::Type{G}=DEFAULT_VECTOR_TYPE) where {
 end
 
 function jacobian_pattern_to_mat(
-    xt::AbstractArray{T}, yt::AbstractArray{<:Number}
+    xt::AbstractArray{T}, yt::AbstractArray{<:Real}
 ) where {T<:GradientTracer}
```

Comment:
Pattern methods need support for `Complex{T}`.

Reply:
#102